Adding basic examples to new-certificate (#902)
OctopusDeploy/docs: docs/octopus-rest-api/octopus.server.exe-command-line/new-certificate.md

      --help                 Show detailed help for this command
```

## Basic examples

> This example will create a new certificate for instance MyNewInstance replacing the old one

Reviewer: ```suggestion

This example creates a new certificate for instance `MyNewInstance` and replaces the old one:
```

      --help                 Show detailed help for this command
```

## Basic examples

This example creates a new certificate for instance `MyNewInstance` and replaces the old one:
```text
octopus.server new-certificate --instance="MyNewInstance" --replace
```

> This example will create a new certificate and export it to a PFX file

Reviewer: ```suggestion
This example creates a new certificate and exports it to a PFX file:
```
